tests/testOptionSlo.php

```diff
@@ -75,15 +75,21 @@ public function testSloFieldOutput() {
 	 */
 	public function testThatSloIsValidatedOnSave() {
 		$validated = self::$admin->basic_validation( [], [ 'singlelogout' => false ] );
-		$this->assertEquals( 0, $validated['singlelogout'] );
+		$this->assertEquals( false, $validated['singlelogout'] );
 
 		$validated = self::$admin->basic_validation( [], [ 'singlelogout' => 0 ] );
-		$this->assertEquals( 0, $validated['singlelogout'] );
+		$this->assertEquals( false, $validated['singlelogout'] );
 
 		$validated = self::$admin->basic_validation( [], [ 'singlelogout' => 1 ] );
-		$this->assertEquals( 1, $validated['singlelogout'] );
+		$this->assertEquals( true, $validated['singlelogout'] );
+
+		$validated = self::$admin->basic_validation( [], [ 'singlelogout' => '1' ] );
+		$this->assertEquals( true, $validated['singlelogout'] );
+
+		$validated = self::$admin->basic_validation( [], [] );
+		$this->assertEquals( false, $validated['singlelogout'] );
 
 		$validated = self::$admin->basic_validation( [], [ 'singlelogout' => uniqid() ] );
-		$this->assertEquals( 1, $validated['singlelogout'] );
+		$this->assertEquals( false, $validated['singlelogout'] );
```

Comment:
@joshcanhelp if 0 is false and 1 is true, this should be true. What am I not seeing?

Reply:
@lbalmaceda - Validation changed do that all truth-y values are not accepted, only specific truth-y values we expected. Previously, some arbitrary string would set this as `true`